tools/data_processing/autocomplete_postprocess.py

```diff
@@ -53,28 +59,33 @@ def traverse_subtree(command, action_dict):
         # Check if the value is a substring of the command
         # hacky way to map spans
         elif type(value) == str and value in command:
-            # Get the span ranges
-            command_word_tokens = command.split(" ")
-            span_tokens = value.split(" ")
-            for i in range(len(command_word_tokens)):
-                if command_word_tokens[i] == span_tokens[0]:
-                    span_start = i
-                    span_end = i
-                    # begin checking for equality
-                    for j in range(1, len(span_tokens)):
-                        if command_word_tokens[i + j] != span_tokens[j]:
-                            break
-                        else:
-                            span_end += 1
-            span_idxs = [0, [span_start, span_end]]
-            action_dict[key] = span_idxs   
+            try:
```

Comment:
should this just be regex?

Reply:
like whether the string is a substring of the command, and that makes it a span?

Reply:
yes- this block of code 62-77

Reply:
I check that already, but how would you get the indices? I try to avoid overly complicated regexes whenever possible